### PR TITLE
Improves the call to action in the frontpage

### PR DESCRIPTION
**Pull request by locks**
Screenshot:

Ember.js home page with new call to action widget

### CODE REVIEW

File: source/about.html.erb [hunk 1]
```diff
@@ -30,23 +30,19 @@ title: "A framework for creating ambitious web applications."
...
+<div class="call-to-action">
+  <div class="top">
+    <h1>Get Started</h1>
+    <div class="cta-copy">
+      <p>Build your first Ember application with our 5 minute tutorial.</p>
+    </div>
 
-  <div class="info">
-    <a href="/builds">More downloads</a>
+    <a href="https://guides.emberjs.com/v2.5.0/getting-started/quick-start/">View Quick Start guide</a>
```
Comment (rwjblue) [inline source/about.html.erb]: This should use the same logic as the guides layout to figure out the version from the JSON file (IMO).

**Comment by rwjblue** [comment]
One nitpick but this looks great!

File: source/about.html.erb (hunk 1 above)
Comment (locks) [inline source/about.html.erb]: 🤔 `versions.json` is in another domain, where do we configure `guides.emberjs.com`'s CORS?

Comment (rwjblue) [inline source/about.html.erb]: It is done in Ruby land at build time, CORS isn't a factor.

**Pull request merged**